app/controllers/volunteers_controller.rb

```diff
@@ -46,6 +47,15 @@ def volunteer_params
       :street, :zip, :city, :nationality, :additional_nationality, :email, :phone, :profession,
       :education, :motivation, :experience, :expectations, :strengths, :skills, :interests, :state,
       :duration, :man, :woman, :family, :kid, :sport, :creative, :music, :culture, :training,
-      :german_course, :adults, :teenagers, :children, :region)
+      :german_course, :adults, :teenagers, :children, :region,
+      language_skills_attributes: [
```

Comment:
you can wrap the language_skills_attributes into a method and move it to a concern and include it in clients and volunteer